.github/workflows/pr-microservice-test.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           aws-region: us-east-1
 
       - name: Set Memory Map Limit
-        if: ${{ contains(matrix.service, "opensearch") }}
+        if: ${{ contains(matrix.service, 'opensearch') }}
         run: sudo sysctl -w vm.max_map_count=262144
```

Comment:
This cmd will not works on the test env, the test is inside an pods created by Action Runner Controller. So I have remove it. And set on the BareMetal of test machine.